recording_server/src/browser.ts

```
@@ -45,16 +50,17 @@ export async function openBrowser(
 
     try {
         console.log('Launching persistent context...')
-
-        // Check that extension path exists
-        const fs = require('fs')
+        
+        // Vérifier que le chemin d'extension existe
+        const fs = require('fs');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)